scripts/deployment/l2/globals_base_mainnet.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"contractVerification":true,"useLedger":true,"derivationPath":"m/44'/60'/2'/0/0","providerName":"base","networkURL":"https://mainnet.base.org","gasPriceInGwei":"2","gnosisSafeAddress":"0x69f4D1788e39c87893C980c06EdF4b7f686e2938","gnosisSafeProxyFactoryAddress":"0xC22834581EbC8527d974F8a1c97E1bEA4EF910BC","baseURI":"https://gateway.autonolas.tech/ipfs/","serviceRegistryName":"Service Registry L2","serviceRegistrySymbol":"AUTONOLAS-SERVICE-L2-V1","bridgeMediatorAddress":"","olasAddress":"0x54330d28ca3357F294334BDC454a032e7f353416","multisigProxyHash130":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000","serviceRegistryAddress":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","operatorWhitelistAddress":"0x3d77596beb0f130a4415df3D2D8232B3d3D31e44","serviceRegistryTokenUtilityAddress":"0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5","serviceManagerTokenAddress":"0x63e66d7ad413C01A7b49C7FF4e3Bb765C4E4bd1b","gnosisSafeMultisigImplementationAddress":"0xBb7e1D6Cb6F243D6bdE81CE92a9f2aFF7Fbe7eac","gnosisSafeSameAddressMultisigImplementationAddress":"0xFbBEc0C8b13B38a9aC0499694A69a10204c5E2aB","stakingTokenAddress":"","stakingNativeTokenAddress":"","minStakingDepositLimit":"1000000000000000000000","timeForEmissionsLimit":"2592000","numServicesLimit":"100","apyLimit":"3000000000000000000","stakingVerifierAddress":"","stakingFactoryAddress":"","livenessRatio":"700000000000000","stakingActivityCheckerAddress":"","stakingParams":{"metadataHash":"","maxNumServices":"100","rewardsPerSecond":"1000000000000000","minStakingDeposit":"50000000000000000000","minNumStakingPeriods":"3","maxNumInactivityPeriods":"3","livenessPeriod":"86400","timeForEmissions":"2592000","numAgentInstances":"1","agentIds":["14"],"threshold":"0","configHash":"0x0000000000000000000000000000000000000000000000000000000000000000","proxyHash":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000","serviceRegistry":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","activityChecker":""}}
```

Comment:
bridgeMediatorAddress also missing?

Reply:
same for celo

Reply:
same for optimism

Reply:
https://basescan.org/address/0xEB5638eefE289691EcE01943f768EDBF96258a80

Reply:
`bridgeMediatorAddress` is irrelevant now

Reply:
https://basescan.org/address/0x10c5525F77F13b28f42c5626240c001c2D57CAd4

Reply:
https://basescan.org/address/0x1cEe30D08943EB58EFF84DD1AB44a6ee6FEff63a

Reply:
https://polygonscan.com/address/0x4aba1Cf7a39a51D75cBa789f5f21cf4882162519

Reply:
https://polygonscan.com/address/0x8Bc0a5c5B1612A16983B01ecd7ae130E728390CB

Reply:
https://polygonscan.com/address/0x46C0D07F55d4F9B5Eed2Fc9680B5953e5fd7b461

Reply:
https://celoscan.io/address/0xe1E1B286EbE95b39F785d8069f2248ae9C41b7a9

Reply:
https://celoscan.io/address/0xc40C79C275F3fA1F3f4c723755C81ED2D53A8D81

Reply:
https://celoscan.io/address/0x1c2cD884127b080F940b7546c1e9aaf525b1FA55

Reply:
https://etherscan.io/address/0x0Dc23eEf3bC64CF3cbd8f9329B57AE4C4f28d5d2

Reply:
https://etherscan.io/address/0x4503b79d468e81Ad2d4bd6Db991810da269bA777

Reply:
https://etherscan.io/address/0xEBdde456EA288b49f7D5975E7659bA1Ccf607efc